src/main/java/com/beerjournal/breweriana/item/rating/persistence/Rating.java

```diff
@@ -0,0 +1,28 @@
+package com.beerjournal.breweriana.item.rating.persistence;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import static lombok.AccessLevel.PRIVATE;
+
+@Document
+@Data
+@EqualsAndHashCode(exclude = "id")
+@RequiredArgsConstructor(access = PRIVATE)
+public final class Rating {
+    @Id
+    private final ObjectId id;
```

Comment:
We shouldn't adjust business to tools, but tools to business, right? :)  
What other solution do you propose?